src/node.cc

```diff
@@ -1994,6 +1994,15 @@ void Hrtime(const FunctionCallbackInfo<Value>& args) {
   args.GetReturnValue().Set(tuple);
 }
 
+void Hrtimefp(const FunctionCallbackInfo<Value>& args) {
+  uint64_t time = 0;
+
+  if (args.Length() > 0)
+    time = args[0]->IntegerValue();
```

Comment:
It seems to me that you truncate the input to whole milliseconds on this line.
Is this intended?